src/test/java/com/exasol/adapter/document/files/SingleDocumentFetcherFactory.java

```diff
@@ -0,0 +1,4 @@
+package com.exasol.adapter.document.files;
```

Comment:
I think this file is not used, please remove if so.

Reply:
:+1: nice catch